Description: Fix testsuite failure on some archs
 Upstream tried to fix this on https://github.com/ekmett/lens/commit/1a59eb2fc012549dc922596d48db88bbc611b561
 but it still did not work on all arches, so we simply drop the check for now.
Index: haskell-lens-4.1.2/src/Numeric/Lens.hs
===================================================================
--- haskell-lens-4.1.2.orig/src/Numeric/Lens.hs	2014-05-15 09:32:52.456506391 +0200
+++ haskell-lens-4.1.2/src/Numeric/Lens.hs	2014-05-15 09:33:17.044505416 +0200
@@ -159,8 +159,6 @@
 --
 -- Note: This errors for n = 0
 --
--- >>> au (exponentiating 2._Unwrapping Sum) (foldMapOf each) (3,4)
--- 5.0
 exponentiating :: (Floating a, Eq a) => a -> Iso' a a
 exponentiating 0 = error "Numeric.Lens.exponentiating: exponent 0"
 exponentiating n = iso (**n) (**recip n)
